src/ResourceManager/Billing/Commands.Billing/Microsoft.Azure.Commands.Billing.Format.ps1xml

```diff
@@ -81,5 +81,27 @@
         </ListEntries>
       </ListControl>
     </View>
+    <View>
+      <Name>Microsoft.Azure.Commands.Billing.Models.PSEnrollmentAccount</Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.Billing.Models.PSEnrollmentAccount</TypeName>
+      </ViewSelectedBy>
+      <ListControl>
```

Comment:
The default should be a table control.  Please provide a table control format here

Reply:
Fixed